src/ast/mod.rs

```diff
@@ -991,8 +991,11 @@ impl Display for WindowType {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
 pub struct WindowSpec {
+    // OVER (PARTITION BY ...)
```

Comment:
If we use `///` then these will show up in documentation as well:

```suggestion
    /// `OVER (PARTITION BY ...)`
```